docs/deployment/README.md

```diff
@@ -214,12 +214,6 @@ $ aws dynamodb create-table \
     --region us-east-1
 ```
 
-### Create a secret for signing Thin Egress App JWTs
```

Comment:
In the template deploy we still have `thin_egress_app` in the cumulus-tf deployment, which needs this. I think it should either be removed from there and the docs updated on how to do a distribution deployment or this should stay.

Reply:
I am thinking have this stay and update the deployment docs to call out the configuration. I am making deployment doc updates so can do that. What do you think?